Intersections_3/include/CGAL/Intersections_3/internal/Segment_3_Segment_3_do_intersect.h

```diff
@@ -30,7 +30,7 @@ do_intersect(const typename K::Segment_3& s1,
 {
   CGAL_precondition(!s1.is_degenerate() && !s2.is_degenerate());
 
-  bool b = do_intersect(s1.supporting_line(), s2.supporting_line(), k);
+  bool b = internal::do_intersect(s1.supporting_line(), s2.supporting_line(), k);
```

Comment:
@afabri We discussed recently and I claimed that we should never call functions unqualified, to avoid ADL.

I just found that item in our coding *rules*:
>  Do not call global functions unqualified, that is, always specify explicitly the namespace where the function is defined. 

See https://doc.cgal.org/latest/Manual/devman_code_format.html#secprogramming_conventions